code/controllers/subsystem/inactivity.dm

```diff
@@ -1,20 +1,25 @@
-#define INACTIVITY_KICK 6000 //10 minutes in ticks (approx.)
+#define INACTIVITY_KICK 10 MINUTES
 
 SUBSYSTEM_DEF(inactivity)
 	name = "Inactivity"
-	wait = INACTIVITY_KICK
+	wait = INACTIVITY_KICK * 0.5
```

Comment:
why * 0.5 ?

Reply:
To clear nulls atleast every 5 minutes. Tg clears them every 6 ticks but even with tens of thousands of runtimes flowing in from the stat panel encountering a mil client, the game didn't grind to a halt.

Reply:
Also worth noting, a delay of 10 minutes firing means that you are giving players 10-20 minutes to be inactive. A fire time of 5 minutes means they can be 10-15 minutes inactive.